clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -119,35 +114,37 @@ func (s *schedulerLatencyMeasurement) getSchedulingLatency(c clientset.Interface
 	}
 
 	for _, sample := range samples {
-		if sample.Metric[model.MetricNameLabel] != schedulingLatencyMetricName {
-			continue
+		var hist *measurementutil.HistogramVec
+		switch sample.Metric[model.MetricNameLabel] {
+		case "scheduler_framework_extension_point_duration_seconds_bucket":
+			switch sample.Metric["extension_point"] {
+			case "Filter":
+				// just record status == Success
+				if sample.Metric["status"] == "Success" {
+					hist = &result.PredicateEvaluationLatency
+				}
+			case "Score":
+				// just record status == Success
+				if sample.Metric["status"] == "Success" {
+					hist = &result.PriorityEvaluationLatency
+				}
+			}
+		case "scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket":
+			hist = &result.PreemptionEvaluationLatency
+		case "scheduler_binding_duration_seconds_bucket":
+			hist = &result.BindingLatency
+		case "scheduler_scheduling_algorithm_duration_seconds_bucket":
+			hist = &result.SchedulingAlgorithmLatency
+		case "scheduler_e2e_scheduling_duration_seconds_bucket":
+			hist = &result.E2eSchedulingLatency
 		}
 
-		var metric *measurementutil.LatencyMetric
```

Comment:
Let's keep the existing summary metrics to not lose history, and add the new ones separately. We can remove the summary metrics once they are removed from scheduler.